src/main/java/com/g4s8/ghman/bot/TkNotifications.java

```diff
@@ -45,19 +44,19 @@
     /**
      * Data source.
```

Comment:
@scristalli please make javadoc correspond new field

Reply:
@olenagerasimova done in commit 0380993